README.md

```diff
@@ -10,7 +10,9 @@ allows using docker containers for functional/integration/~~unit~~ testing.
 
 > TestContainers is a Java 8 library that supports JUnit tests, providing lightweight, throwaway instances of common databases, Selenium web browsers, or anything else that can run in a Docker container.
 
-Testcontainers-scala in action: http://dimafeng.com/2016/08/01/testcontainers-selenium/
+[testcontainers-scala in action](http://dimafeng.com/2016/08/01/testcontainers-selenium)
+
+[SBT Microsite](https://testcontainers.github.io/testcontainers-scala)
```

Comment:
Unfortunately, I didn't have a chance to make the microsite work. I think it'd be better to not link to it to not mislead people.